oracle/src/events/processCollectedSignatures/index.js

```diff
@@ -46,6 +47,34 @@ function processCollectedSignaturesBuilder(config) {
         logger.info(`Processing CollectedSignatures ${colSignature.transactionHash}`)
         const message = await homeBridge.methods.message(messageHash).call()
 
+        if (config.accessLists) {
+          const parsedMessage = parseMessage(message)
+          const recipient = parsedMessage.recipient.toLowerCase()
+          const originalTxHash = parsedMessage.txHash
+          const allowanceList = await readAccessListFile(config.accessLists.allowanceList)
+          const blockList = await readAccessListFile(config.accessLists.blockList)
+
+          if (allowanceList.length > 0) {
+            const sender = (await web3Home.eth.getTransaction(originalTxHash)).from.toLowerCase()
```

Comment:
I would suggest an optimization here: to check the sender only If the recipient is not found in the list. In most cases it will be enough since only few users understand how to use the alternative recipient feature.

Reply:
Sure, makes sense.